### DISCUSSION TITLE
[@types/node] https.RequestOptions lacks checkServerIdentity property

### DISCUSSION THREAD

**Discussion by lgrahl** [DefinitelyTyped/DefinitelyTyped#55949] [Issues with a @types package]
`https.RequestOptions` lacks the `checkServerIdentity` property but is mentioned in an example in https://nodejs.org/dist/latest-v14.x/docs/api/https.html#https_https_request_url_options_callback

Likewise `tls.PeerCertificate` lacks the `pubkey` property (same example).

[a comment by an automated account is not shown]

**Reply by fgblomqvist** [reply]
What are the next steps here? It's still missing `checkServerIdentity`. Should I open a PR?

**Reply by peterblazejewicz** [reply]
@fgblomqvist yes, please. That's the next step in open source, thanks!

**Reply by Semigradsky** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63053
